package.json

```diff
@@ -3,7 +3,11 @@
   "version": "0.0.1",
   "author": "Sam Curren <telegramsam@gmail.com>",
   "description": "Tools for working with Aries Agents",
-  "license": null,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:hyperledger/aries-toolbox.git"
```

Comment:
Might be better to use `https://github.com/hyperledger/aries-toolbox.git`

Reply:
I agree.